src/app/domain/authentication/services/AuthService.ts

```diff
@@ -0,0 +1,49 @@
+import Minilog from '@cozy/minilog'
+
+import type CozyClient from 'cozy-client'
+
+import { clearClient } from '/libs/clientHelpers/persistClient'
+import { resetSessionToken } from '/libs/functions/session'
+import { deleteKeychain } from '/libs/keychain'
+import { clearCookies } from '/libs/httpserver/httpCookieManager'
+import { clearData } from '/libs/localStore/storage'
+import { routes } from '/constants/routes'
+import { navigationRef } from '/libs/RootNavigation'
+
+export const authLogger = Minilog('[AuthService]')
+let clientInstance: CozyClient | null = null
+
+const asyncLogoutNoClient = async (): Promise<void> => {
```

Comment:
nit: it would be great to move this code near `asyncLogout` so we can easily see that both version exist and we don't forget to edit both if we add an item to clear on logout.

Or event better: find a way to mutualize them.

Reply:
yeah but asyncLogout isn't in the greatest place right now. I have to check if it's easy to move it now